.travis.yml

```diff
@@ -18,17 +20,22 @@ install:
   - wait-on http://localhost:7474 && docker-compose exec neo4j migrate
 
 script:
+  # Backend
   - docker-compose exec backend yarn run lint
   - docker-compose exec backend yarn run test:jest --ci --verbose=false
   - docker-compose exec backend yarn run db:reset
   - docker-compose exec backend yarn run db:seed
   - docker-compose exec backend yarn run test:cucumber
   - docker-compose exec backend yarn run db:reset
   - docker-compose exec backend yarn run db:seed
+  # Frontend
   - docker-compose exec webapp yarn run lint
   - docker-compose exec webapp yarn run test --ci --verbose=false
   - docker-compose exec -d backend yarn run test:before:seeder
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-08T13:58:37Z" title="Wednesday, May 8th 2019, 3:58:37 pm +02:00">May 8, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -18,17 +20,22 @@ install:
   - wait-on http://localhost:7474 && docker-compose exec neo4j migrate
 
 script:
+  # Backend
   - docker-compose exec backend yarn run lint
   - docker-compose exec backend yarn run test:jest --ci --verbose=false
   - docker-compose exec backend yarn run db:reset
   - docker-compose exec backend yarn run db:seed
   - docker-compose exec backend yarn run test:cucumber
   - docker-compose exec backend yarn run db:reset
   - docker-compose exec backend yarn run db:seed
+  # Frontend
   - docker-compose exec webapp yarn run lint
   - docker-compose exec webapp yarn run test --ci --verbose=false
   - docker-compose exec -d backend yarn run test:before:seeder
```


That odd line here is by the way a prerequisite of fullstack testing. Cypress needs to setup data. Therefore we use the same method as in the backend tests: We have a backend server with certain permissions disabled to setup data when we need it. This line here starts the seeder/factory server.

If you want to improve the code comment, you can swap lines 35 <=> 34.

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-05-08T14:05:03Z" title="Wednesday, May 8th 2019, 4:05:03 pm +02:00">May 8, 2019</time>_

---


Not in scope of this pr